distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Named_Pattern.enso

```diff
@@ -0,0 +1,34 @@
+from Standard.Base import all
+
+## PRIVATE
+type Named_Pattern
+   Duplicate_Whitespace
+   Leading_Whitespace
+   Trailing_Whitespace
+   All_Whitespace
+   Leading_Numbers
+   Trailing_Numbers
+   Non_ASCII
+   Tabs
+   Letters
+   Numbers
+   Punctuation
+   Symbols
```

Comment:
Please add the doc comments (I already had added them back here).

Reply:
I made the type private as I don't think it should show up in the CB outside of where we use it.

Reply:
Done